common/genes/03_genes_accessories_beards.txt

```diff
@@ -303,5 +303,20 @@ accessory_genes = {
 				1 = empty
 			}
 		}
+		ogre_beards = {
+			index = 15
+			male = {
+				15 = male_beard_indian_03 #
+				5 = male_beard_steppe_02
+				15 = male_beard_northern_03
+				15 = male_beard_northern_02
+				15 = male_beard_northern_01
+				15 = male_beard_western_07
+				5 = male_beard_mena_03
+			}
+			female = {
+				1 = empty
+			}
+		}
```

Comment:
These are orcish beards except `male_beard_steppe_02`. Maybe we don't need this gene and the ogres will use the orc beards?
What to do with `male_beard_steppe_02` is up to you. You may include it in the orcish beards.

Reply:
> 
> 
> These are orcish beards except `male_beard_steppe_02`. Maybe we don't need this gene and the ogres will use the orc beards?
> What to do with `male_beard_steppe_02` is up to you. You may include it in the orcish beards.

I added it for a reason. Ogres are canonically supposed to be able to grow hair on their upper lip as well, so I wanted to have one beard like that. As you can see, I made it much rarer to occur than the other beards. Not to mention ogres having a beard, in general, is quite rare.

But I suppose we could just use the orc beards and add `male_beard_steppe_02` into the mix as a rare alternative.

Reply:
Addressed this in f7b06f9492a29674102d6c15e20dc8c9f1a4de7b.